scripts/drs_file_access_examination.ipynb

```diff
@@ -0,0 +1,326 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Data Repository Service (DRS) DRS File Access Examination\n",
+    "\n",
+    "\n",
+    "In this notebook, we get all the DRS urls from the data table and check if we can access the file each DRS url points to.\n",
+    "\n",
+    "This notebook will:\n",
+    "1. Grab all DRS urls from the data table\n",
+    "2. Resolve each DRS url and check our access to the file\n",
+    "3. Create a report with all the DRS urls and our access level to each of them\n",
+    "\n",
+    "## Background\n",
+    "\n",
+    "\n",
+    "The Data Repository Service (DRS) API is a standardized set of access methods that are agnostic to cloud infrastructure. Developed by the Global Alliance for Genomics and Health (GA4GH), DRS enable researchers to access data regardless of the underlying architecture of the repository (i.e. Google Cloud, Azure, AWS, etc.) in which it is stored. Terra supports accessing data using the GA4GH standard Data Repository Service (DRS). To learn more look at this link: https://support.terra.bio/hc/en-us/articles/360039330211-Data-Access-with-the-GA4GH-Data-Repository-Service-DRS- \n",
+    "\n",
+    "## Setup\n",
+    "\n",
+    "To run code from the terra-notebook-utils DRS library, we first have to install the package and restart the kernel. \n",
+    "\n",
+    "1. Run the cell below\n",
+    "2. Go to the kernel tab in the menu at the top, and select restart\n",
+    "3. Press the restart in the pop-up menu"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Installing terra-notebook-utils library\n",
+    "!pip install terra-notebook-utils"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Grab all DRS urls from the data table\n",
+    "\n",
+    "The DRS url is stored under the object_id column in a table.\n",
+    "\n",
+    "Using the firecloud library, we are going to search for the object_id column in each table in the data table. If a table contains an object_id column, we check if the content is a string and has 'drs://' at its beginning."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "code_folding": [],
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from firecloud import api as fapi\n",
+    "\n",
+    "# Getting the namespace/project and name of the workspace\n",
+    "namespace = os.environ['WORKSPACE_NAMESPACE']\n",
+    "workspace = os.environ['WORKSPACE_NAME']\n",
+    "\n",
+    "# Getting list of tables in the data table \n",
+    "data_tables_list = fapi.get_entities_with_type(namespace, workspace).json()\n",
+    "\n",
+    "# Assigning column search key\n",
```

Comment:
add a workspace identifying option that determines the column_name_search_key (based on our team discussion)